docs/extensions/api.md

```diff
@@ -807,8 +837,8 @@ module and call the `makeExtension` function to initiate your extension.
 [makeExtension]: #makeextension
 [ElementTree]: https://effbot.org/zone/element-index.htm
 [Available Extensions]: index.md
-[Footnotes]: https://github.com/Python-Markdown/mdx_footnotes
-[Definition Lists]: https://github.com/Python-Markdown/mdx_definition_lists
+[Footnotes]: https://github.com/Python-Markdown/markdown/blob/master/markdown/extensions/footnotes.py
+[Definition Lists]: https://github.com/Python-Markdown/markdown/blob/master/markdown/extensions/definition_lists
```

Comment:
Why were these links changed? They were linking to the documentation for the extensions. That's what we want.

I can see the benefit of helping people find the source code, but that should be a secondary link. Perhaps something like: `... [Footnotes Extension][footnotes] (see [source code][footnotes_src])...`

Reply:
The old links are 404…

Reply:
> The old links are 404…

Fair point. I was going to say that they used to point to documentation, but looking at them again, I don't know what they pointed to. I don't think those URL's were ever valid. I'd have to check the history to see what they pointed to.